### DISCUSSION TITLE
How to download a release from a private repo using wget or curl?

### DISCUSSION THREAD

**Discussion by marmayogi** [community/community#36647] [Other Feature Feedback, Questions, & Ideas]
I Just made a `release` whose repository is `private`. Anybody can download such a release using `wget` or `curl` command.
My question is should anybody need personal token(authorization) associated with private repo in order to download the `tar` files using `wget` or `curl` command?

My question may be basic one but I need a solid answer.

Thanks in advance.

**Comment by notthetup** [comment]
https://gist.github.com/maxim/6e15aa45ba010ab030c4
